libs/ballot-interpreter/README.md

```diff
@@ -183,23 +203,78 @@ etc.
 
 ### Score Bubble Marks
 
-Bubble marks are scored to determine which bubbles are filled in. Locating the
-bubble at `(x, y)` first locates the `y`th timing mark on the left and right
-sides of the ballot. Given a line segment between the centers of these two
-timing marks, the center of the bubble is presumed to be on this line `x / N`
-percent of the way from the left timing mark to the right timing mark, where `N`
-is the number of columns of timing marks on the ballot. The bubble is then
-scored by comparing a template bubble image to the actual contents of the scan
-at the bubble location. A score is computed by computing the number of new dark
-pixels compared to the template bubble image. The score is later compared to a
-threshold to determine if the bubble is filled in, but the core function simply
+Bubble marks are scored to determine which bubbles are filled in. The scoring
+process ([scoring.rs](src/bubble-ballot-rust/scoring.rs)) consists of two main
+steps: template matching and fill scoring.
```

Comment:
This sentence is missing the "Bubble Localization" step. Also "localization" is kind of a weird word because it makes me think of language translation

Reply:
Changed to “Bubble Locating”.